GUI/QtQuick/qml/Settings/SettingsPopup.qml

```diff
@@ -101,7 +102,11 @@ CustomPopup {
         settings_menu.init()
     }
 
-    Component.onCompleted: {
-        root.armoryServersModel = bsApp.getArmoryServers()
+    function updateServersModel()
+    {
+        if (root.armoryServersModel === null)
+        {
+            root.armoryServersModel = bsApp.getArmoryServers()
```

Comment:
The model should be re-created on each update (`getArmoryServers` sends API request)